OpenEXR/IlmImf/ImfFastHuf.cpp

```diff
@@ -255,7 +255,7 @@ FastHufDecoder::FastHufDecoder
         int codeLen = *i & 63;
         int symbol  = *i >> 6;
 
-        if (mapping[codeLen] >= _numSymbols)
+        if (mapping[codeLen] >= Int64(_numSymbols))
```

Comment:
static_cast<Int64>

Reply:
I've converted all constructor casts to static_cast